src/main/java/core/basesyntax/transaction/TransactionHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.transaction;
+
+public interface TransactionHandler {
```

Comment:
Better to rename
```suggestion
public interface OperationHandler {
```
and inherited classes should be renamed too